packages/langium/src/utils/file-utils.ts

```diff
@@ -0,0 +1,18 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import fs from 'fs-extra';
+import path from 'path';
+
+export function buildRealpath(pathToRepoPackage: string, packageFile: string): string {
```

Comment:
Do you use this in the actual `langium` package? If not, this probably belongs to the `test` directory.

Reply:
Wasn't sure. Do you see further need for this beyond the tests?

Reply:
It will go away. See https://github.com/langium/langium/pull/179#discussion_r673150611

Reply:
Not right now, but I also can't envision a use case where this becomes necessary. I'm especially concerned with the hardcoded stuff in there, at least in the `buildRealpath` method. The other method is fine, imo.